Runner/suites/Kernel/FunctionalArea/baseport/USBHost/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+#Setup requires at least one USB peripheral connected to USB port that supports Host mode function
+
+# Import test suite definitions
+. $(pwd)/init_env
```

Comment:
please add quotes here to prevent word splitting. 
```
. "${PWD}"/init_env
```
This is the last bit before merging. The rest looks good!